src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/proxy.conf.js

```diff
@@ -1,23 +0,0 @@
-const { env } = require('process');
-
-const target = env.ASPNETCORE_HTTPS_PORT ? `https://localhost:${env.ASPNETCORE_HTTPS_PORT}` :
-  env.ASPNETCORE_URLS ? env.ASPNETCORE_URLS.split(';')[0] : 'http://localhost:8080';
-
-const PROXY_CONFIG = [
-  {
-    context: [
-      "/weatherforecast",
-//#if (IndividualLocalAuth)
-      "/_configuration",
-      "/.well-known",
-      "/Identity",
-      "/connect",
-      "/ApplyDatabaseMigrations",
-//#endif
-   ],
-    target: target,
-    secure: false
-  }
-]
-
-module.exports = PROXY_CONFIG;
```

Comment:
Removing this file is wrong. This is needed to proxy to the backend

Reply:
I misinterpreted your comment about this in angular.json then, so just delete the entry in angular.json and leave the file?

Reply:
Yep, only the entry on the production bit. On development we still need the entry, and we need the file.